stan/math/prim/mat/prob/ordered_logistic_glm_lpmf.hpp

```diff
@@ -75,8 +75,12 @@ ordered_logistic_glm_lpmf(
   check_consistent_size(function, "Weight vector", beta, N_attributes);
   check_bounded(function, "Vector of dependent variables", y, 1, N_classes);
   check_ordered(function, "Cut-points", cuts);
-  check_finite(function, "Final cut-point", cuts[N_classes - 2]);
-  check_finite(function, "First cut-point", cuts[0]);
+  if (N_classes > 1) {
+    if (N_classes > 2) {
+      check_finite(function, "Final cut-point", cuts[N_classes - 2]);
+    }
+    check_finite(function, "First cut-point", cuts[0]);
+  }
```

Comment:
Can you elaborate on this change. I dont think its mentioned in the PR text.

Reply:
This was a bug. Thise checks could cause segfault if number of classes was less than 2.